src/adapter/src/coord/timeline.rs

```diff
@@ -333,22 +333,28 @@ impl Coordinator {
         let mut id_bundle = CollectionIdBundle::default();
         for entry in self.catalog().entries() {
             if let TimelineContext::TimelineDependent(entry_timeline) =
-                self.get_timeline_context(entry.id())
+                self.get_timeline_context(entry.latest_global_id())
```

Comment:
It feels like timeline stuff should only care about catalog IDs, not global IDs. Are we envisioning that you could ever move an object from one timeline to another?

Reply:
Yeah good call. I originally left the timeline APIs using `GlobalId` since a lot of the callers of these methods use `GlobalId`s, but refactored it to use `CatalogItemId`